src/test/java/org/zendesk/client/v2/RealSmokeTest.java

```diff
@@ -315,9 +315,9 @@ public void getTriggersWithParameters() throws Exception {
     createClientWithTokenOrPassword(2);
     int count = 0;
     String title = null;
-    for (Trigger t : instance.getTriggers(null, true, "title", SortOrder.ASCENDING)) {
+    for (Trigger t : instance.getTriggers(null, true, "alphabetical", SortOrder.ASCENDING)) {
       if (title != null) {
-        assertTrue(title.compareTo(t.getTitle()) < 0);
```

Comment:
The comparison should ignore case (seen this while running against our test instance).